tap_stripe/streams.py

```diff
@@ -580,6 +581,7 @@ class Products(StripeStreamV2):
     name = "products"
     replication_key = "updated"
     object = "product"
+    event_filter = "product.*"
```

Comment:
same as plans for products we need to get events from `products.* `and from `invoice.*`